pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -22,7 +22,7 @@ from numpy cimport (
 
 cnp.import_array()
 
-from cpython.datetime cimport (  # alias bc `tzinfo` is a kwarg below
+from cpython.datetime cimport (  # alias tzinfo_type bc `tzinfo` is a kwarg below
```

Comment:
hmm i had tried to put this comment adjacent to the tzinfo_type import but looks like one of the hooks moved it back